### DISCUSSION TITLE
اختیار معامله

### DISCUSSION THREAD

**Discussion by odkeihancpu** [ARahimiQuant/finpy-tse#9] [General]
سلام دوست من خداقوت 
میتونید ترتیبی فراهم کنید که اطلاعات مربوط به قراردادهای اختیار ها رو هم بشه گرفت. و یک سوال دیگه اینکه خود این کتابخانه این اطلاعات رو چطوری دریافت میکنه ؟ APi? اسکرپینگ

**Comment by ARahimiQuant** [comment]
سلام و پوزش بابت تاخیر در پاسخگویی،

اطلاعات قراردادهای اختیار شامل قیمت و حقیقی حقوقی رو میتونید با همون توابعی که برای سهم ها استفاده میکنید بگیرید، به شرطی که اسم رو دقیق وارد کنید. در مورد سایر اطلاعات سعی خواهیم کرد در نسخه های بعدی اضافه کنیم. کتابخانه اطلاعات رو از طریق وبکاوی جمع آوری میکنه.
